### ISSUE TITLE
/input set_unread doesn't sync

### ISSUE THREAD

**Issue by devkev** [wee-slack/wee-slack#323]
The `/input set_unread_current_buffer` (and `/input set_unread`) commands will mark the buffer as being read in weechat, but does not sync this change to Slack.

See also #307 and #319.

**Comment by devkev** [comment]
I was able to work around this with

```
/trigger add update_slack_when_mark_read command_run "/input set_unread_current_buffer"
/trigger set update_slack_when_mark_read conditions ${plugin} == python && ${script_name} == slack_extension
/trigger set update_slack_when_mark_read command /slack markread
```

But this is a hack (ie. potentially racy), so it would still be better if wee-slack itself hooked into `/input set_unread_current_buffer` and `/input set_unread` as being the canonical way of marking buffers read.  Then `/slack markread` could just be an alias for `/input set_unread_current_buffer`.  `/slack setallreadmarkers` could maybe remain, as a way of marking all slack buffers as read (and not irc buffers, for instance).  This would let existing weechat integrations that use these commands (as well as the default weechat keybindings) work as expected.

**Comment by trygveaa** [comment]
This is now implemented in #441.
